roles/setup_ntp/templates/chrony.conf.j2

```diff
@@ -1,20 +1,23 @@
 # {{ ansible_managed }}
-stratumweight 0
 driftfile /var/lib/chrony/drift
-rtcsync
-makestep 10 3
+bindcmdaddress {{ ntp_server }}
 bindcmdaddress 127.0.0.1
 bindcmdaddress ::1
 keyfile /etc/chrony.keys
-noclientlog
-logchange 0.5
-logdir /var/log/chrony
-manual
 local stratum 10
+rtcsync
+makestep 1.0 3
+manual
+
+logdir /var/log/chrony
+log measurements statistics tracking
```

Comment:
Do we want to add this for all setups?

Reply:
This only applies to the ntp_host. And yes it will automatically set the listener to to the same address that we set the ntp_host to be.